src/components/task.js

```diff
@@ -1,47 +1,59 @@
-export const createTaskTemplate = () => {
-  return (
-    `<article class="card card--black">
-      <div class="card__form">
-        <div class="card__inner">
-          <div class="card__control">
-            <button type="button" class="card__btn card__btn--edit">
-              edit
-            </button>
-            <button type="button" class="card__btn card__btn--archive">
-              archive
-            </button>
-            <button
-              type="button"
-              class="card__btn card__btn--favorites card__btn--disabled"
-            >
-              favorites
-            </button>
-          </div>
+import {MONTH_NAMES} from "../const.js";
+import {formatTime} from "../utils.js";
 
-          <div class="card__color-bar">
-            <svg class="card__color-bar-wave" width="100%" height="10">
-              <use xlink:href="#wave"></use>
-            </svg>
-          </div>
 
-          <div class="card__textarea-wrap">
-            <p class="card__text">Example default task with default color.</p>
-          </div>
+export const createTaskTemplate = (task) => {
+  const {description, dueDate, color, repeatingDays, isArchive, isFavorite} = task;
```

Comment:
Если аргумент `task` вам нужен только для декомпозиции, можно сократить
```suggestion
export const createTaskTemplate = ({description, dueDate, color, repeatingDays, isArchive, isFavorite}) => {
```